.github/workflows/ms.compute.disks.yml

```
@@ -0,0 +1,152 @@
+name: 'Compute: Disks'
+
+on:
+  workflow_dispatch:
+    inputs:
+      removeDeployment:
+        type: boolean
+        description: 'Remove deployed module'
+        required: false
+        default: 'true'
+      versioningOption:
+        type: choice
+        description: 'The mode to handle the version increments [major|minor|patch]'
+        required: false
+        default: 'patch'
+        options:
+          - major
+          - minor
+          - patch
+      customVersion:
+        description: 'Custom version to apply. Used only if higher than latest'
+        required: false
+        default: '0.0.1'
+  push:
+    branches: [main, users/masanzde/disks-module]
```

Sorry for noticing only just now, please remove `users/masanzde/disks-module`
```suggestion
    branches: 
      - main
```

---

good catch! fixed :)